content/en/docs/concepts/workloads/controllers/replicaset.md

```diff
@@ -223,7 +223,7 @@ In this manner, a ReplicaSet can own a non-homogenous set of Pods
 
 As with all other Kubernetes API objects, a ReplicaSet needs the `apiVersion`, `kind`, and `metadata` fields.
 For ReplicaSets, the `kind` is always a ReplicaSet.
-In Kubernetes 1.9 the API version `apps/v1` on the ReplicaSet kind is the current version and is enabled by default. The API version `apps/v1beta2` is deprecated.
+The API version `apps/v1` on the ReplicaSet kind is the current version and is enabled by default.
```

Comment:
This API is stable enough that you could even remove this line as well.

Reply:
Thanks @sftim. Removed it and the subsequent line a it would be irrelevant.